### PR TITLE
T1 76.fix font styles

### PR DESCRIPTION
**Pull request by diegoplatap**
# Improve register and add pet stlyes

- We found there were some inconsistencies with the font size in the `RegisterPage.js`, now is fixed.
- We founf there were some inconsistencies with the color of the `AddPet.js` inputs.
- Add waves to the  `AddPet.js` page.
- Update the size of the button and styles depending of when is used.

image
image
image

### CODE REVIEW

File: src/assets/styles/PrimaryButton.css [hunk 1]
```diff
@@ -1,3 +1,49 @@
+html {
+  font-size: 10px;
+}
+
+.registerForm{
```
Comment (leoneloliveros) [inline src/assets/styles/PrimaryButton.css:5]: RegisterForm, addPet and updateUser are almost the same. create one selector add attributes in common

File: src/assets/styles/PrimaryButton.css [hunk 2]
```diff
@@ -26,4 +72,40 @@
...
   width: 80%;
+}
+
+@media (min-width: 768px) {
+  .registerForm{
+    width: 60%
+  } 
+}
+
+@media (min-width: 1024px) {
```
Comment (leoneloliveros) [inline src/assets/styles/PrimaryButton.css:83]: Media query 768 and 1024 are duplicated use one

File: src/assets/styles/PrimaryButton.css (hunk 1 above)
Comment (diegoplatap) [reply src/assets/styles/PrimaryButton.css]: Thanks leonel. I refactor and now there is less noise in the code. No need for that duplicated styles in the classes!

File: src/assets/styles/PrimaryButton.css (hunk 2 above)
Comment (diegoplatap) [reply src/assets/styles/PrimaryButton.css]: Thanks leonel. I refactor the mediaqueries duplication. No need for that much code to obtain same results.

**Review by leoneloliveros — APPROVED** [review]

**Pull request merged**